config/example.yaml

```diff
@@ -4,8 +4,18 @@ regions:
 account-blacklist:
 - 1234567890
 
+resource-types:
+  target:
```

Comment:
Consider having two examples, one with a blacklist and another one with a whitelist.

Reply:
fxeid